### ISSUE TITLE
AutoItLibrary must be installed on C: drive

### ISSUE THREAD

**Issue by GoogleCodeExporter** [zhangbignzhen/robotframework-autoitlibrary#12]
```
What steps will reproduce the problem?
1.I install in this order :
   active python 2.5.5.7
   robotframework2.1.3
   autiItlibrary 1.0

What is the expected output? What do you see instead?
When I launch the test provided into the autoitlibrary (RunTests.bat), an 
error occur: No keyword with name 'Send' found; No keyword with name 'Win 
Activate' found.
I haven't found how these keyworg are defined.
I haven't error during the installation.
The test is installed on C:\Robotframework\Extensions\AutoItLibrary.
Robotframework is installed on D:\python25

What version of the product are you using? On what operating system?
ActivePython-2.5.5.7-win32-x86.msi
robotframework-2[1].1.3.win32.exe
AutoItLibrary-1[1].0.zip

Please provide any additional information below.

Thanks for your help.
```

Original issue reported on code.google.com by `vTB...@gmail.com` on 22 Apr 2010 at 3:30

**Comment by GoogleCodeExporter** [comment]
```
Please attach the report.html and log.html from your Robot Framework test run.  
I
suspect that this is due to hard coding of "C:\..." in the AutoItLibrary.  That 
is
already fixed in my version and planned to be released soon.  In the mean time 
you
could try moving your Python and all its packages from D:\python25 to 
C:\python25
(best to uninstall and re-install so the automatic path setting gets changed to 
the
new location).
```

Original comment by `c.martin...@gmail.com` on 22 Apr 2010 at 5:53
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
You are right. It's OK.

I uninstall all and I install python on disk C:.

The test passed.

Thanks.

```

Original comment by `vTB...@gmail.com` on 23 Apr 2010 at 10:37

**Comment by GoogleCodeExporter** [comment]
Original comment by `c.martin...@gmail.com` on 23 Apr 2010 at 11:34
- Changed title: **AutoItLibrary must be installed on C: drive**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with solution to Issue #9.
```

Original comment by `c.martin...@gmail.com` on 23 Apr 2010 at 8:03
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `c.martin...@gmail.com` on 23 Apr 2010 at 8:08
- Added labels: **Milestone-Release1.1**
